cost/kubecost/sizing/kubecost_resizing_recommendation.pt

```diff
@@ -137,31 +146,83 @@ datasource "ds_get_currency" do
   end
 end
 
-script "js_request_sizing", type: "javascript" do
-  parameters "kubecost_host", "param_cpu_algo", "param_cpu_q", "param_mem_algo", "param_mem_q", "param_window", "param_cpu_util", "param_mem_util"
+datasource "ds_get_clusters" do
+  request do
+    verb "GET"
+    scheme "http"
+    host $param_kubecost_host
+    path "/model/clusterInfoMap"
+  end
+  result do
+    encoding "json"
+    collect jmes_path(response, "data | values(@)") do
+      field "clusterId", jmes_path(col_item, "id")
+    end
+  end
+end
+
+datasource "ds_sizing_recommendation_request_list" do
+  run_script $js_sizing_recommendation_request_list, $param_kubecost_host, $param_recommendations_request_method, $ds_get_clusters, $param_cpu_algo, $param_cpu_q, $param_mem_algo, $param_mem_q, $param_window, $param_cpu_util, $param_mem_util
+end
+
+script "js_get_request_from_iter_item", type: "javascript" do
```

Comment:
I think each script should be next to data source where it's used, more info here: https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices#Scripts.